src/include/txn_inline.h

```diff
@@ -797,7 +797,8 @@ __txn_visible_id(WT_SESSION_IMPL *session, uint64_t id)
  *     Can the current transaction see the given ID / timestamp?
  */
 static inline bool
-__wt_txn_visible(WT_SESSION_IMPL *session, uint64_t id, wt_timestamp_t timestamp)
+__wt_txn_visible(
```

Comment:
Don't we need to change the wt_txn_visible_all as well ??

Reply:
Other than the metadata table, we always use the durable timestamp passed to the `wt_txn_visible_all` function.

Reply:
Currently, we do not have the WT_TIME_POINT as mentioned by Alex, to add a check against future changes, I believe it is better to handle it when the WT_TIME_POINT structure is added. Opinions?

Reply:
I'm in favour of that.

Reply:
sgtm

Reply:
Probably worth adding a link to this comment in the ticket that will add the new structure or add details to the scope in the ticket's description (even better)

Reply:
Thanks, Etienne. I created the WT-10612 ticket and linked this PR comment also.